libnmstate/varlink/nmstate-varlink@.service

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# This file is part of nmstate
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+
+[Unit]
+Description= Nmstate's libnmstate varlink service
+Documentation=https://www.nmstate.io/devel/libnmstate-varlink.html
+After=multi-user.target
+Requres=NetworkManager.service
+ConditionPathExists=/usr/bin/io.nmstate
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/io.nmstate %i
```

Comment:
Only unix socket is allowed here. Due to privilege leak when using tcp socket.

You may take `/usr/lib/systemd/system/io.podman.service` as reference.

And there is no need to run nmstate as non-root user. Just make it root only.